### PR TITLE
[ADD] GE1: Motorcycle Filter

### PR DESCRIPTION
**Pull request by vmac-odoo**
## Features
- Motorcycle Filter Generated: shows only motorcycle type products when is activated
- Default Filter: filter set as default on init

### CODE REVIEW

File: ge01_team02/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "GE01: Motorcycle Filter",
+    "summary": "Filter Motorcycle",
+    "description": """
+    Motorcycle GE01_team02""",
+    "version": "0.1",
```
Comment (ayhp-odoo) [inline ge01_team02/__manifest__.py:6]: The version should be 1.0.0

File: ge01_team02/views/product_template_search_inherit.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_search_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//filter[@name='consumable']" position="after">
+                    <filter string="Motorcycle" name="filter_motorcycle" domain="[('detailed_type', '=', 'motorcycle')]"/>
+                </xpath>
+            </field>
+        </record>
+    </data>
+</odoo>
```
Comment (ayhp-odoo) [inline ge01_team02/views/product_template_search_inherit.xml:19]: Add a new line at the end of the file

**Review by ayhp-odoo — CHANGES_REQUESTED** [review]

**Comment by ayhp-odoo** [comment]
Please read the GitHub guidelines of Odoo regarding commits and Pull Request, is necessary to explain what does this changes will do.

https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html?highlight=guidelines#configure-your-git

File: ge01_team02/views/product_template_search_inherit.xml (hunk 2 above)
Comment (vmac-odoo) [reply ge01_team02/views/product_template_search_inherit.xml:17]: Solved Issue

File: ge01_team02/views/product_template_search_inherit.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
[the remark this answers is not in the document]
Comment (vmac-odoo) [reply ge01_team02/views/product_template_search_inherit.xml]: Solved Issue

File: ge01_team02/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "category": "Kauil/Registry",
+    "license": "OPL-1",
+    "depends": ["motorcycle_registry", "sale"],
```
[the remark this answers is not in the document]
Comment (vmac-odoo) [reply ge01_team02/__manifest__.py]: Solved Issue

File: ge01_team02/views/product_template_search_inherit.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    <record id="sale.product_template_action" model="ir.actions.act_window">
+        <field name="context">{"search_default_filter_motorcycle":1}</field>
+        <field name="search_view_id" ref="product.product_template_search_view"/>
+    </record>
+    <record id="sale_order_search_inherit" model="ir.ui.view">
+            <field name="name">product.template.search.inherit.ge01_team02</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_search_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//filter[@name='consumable']" position="after">
```
Comment (ayhp-odoo) [inline ge01_team02/views/product_template_search_inherit.xml:12]: This is good but also you can do it as this:

`<filter name="consumable"  position="after">`

**Review by ayhp-odoo — CHANGES_REQUESTED** [review]

File: ge01_team02/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "GE01: Motorcycle Filter",
+    "summary": "Filter Motorcycle",
+    "description": """
+    Motorcycle GE01_team02""",
```
[the remark this answers is not in the document]
Comment (vmac-odoo) [reply ge01_team02/__manifest__.py]: Solved

File: ge01_team02/views/product_template_search_inherit.xml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+                    <filter string="Motorcycle" name="filter_motorcycle" domain="[('detailed_type', '=', 'motorcycle')]"/>
+                </xpath>
+            </field>
+        </record>
+</odoo>
```
[the remark this answers is not in the document]
Comment (vmac-odoo) [reply ge01_team02/views/product_template_search_inherit.xml]: Solved

File: ge01_team02/views/product_template_search_inherit.xml (hunk 5 above)
Comment (vmac-odoo) [reply ge01_team02/views/product_template_search_inherit.xml]: Solved

**Review by ayhp-odoo — APPROVED** [review]

**Pull request merged**